telemetry/telemetry.go

```diff
@@ -268,7 +268,7 @@ func Failure(message string) {
 // Report success of an operation
 func Success(message string) {
 	analyticsClient.Track(distinctID(),
-		message,
+		fmt.Sprintf("Success in %s", message),
```

Comment:
For example, in this case we have only two instances:

"Success in rotate learn session"
"Success in Add to ECS"

which is inconsistent capitalization :) and I don't think the success adds much?